### ISSUE TITLE
package-appclient.cmd -xml my-acc.xml creates entry "appclient/file:" in appclient.jar

### ISSUE THREAD

**Issue by mkarg** [payara/Payara#1828]
# Description #
----------
When using the (apparently undocumented) option `-xml my-acc.xml` of `package-appclient.cmd` on windows, an entry named `file:` is created in the root of the resulting appclient.jar file.

## Expected Outcome
The created appclient.jar contains entry `appclient/glassfish/domains/domain1/config/glassfish-acc.xml` which is a copy of my-acc.xml.

## Current Outcome
The created appclient.jar contains entry `appclient/file:/C:/Users/Markus%20Karg/x/my-acc.xml` which is a copy of my-acc.xml.

## Steps to reproduce (Only for bug reports) 
```batch
mkdir "C:\Users\Markus Karg\x"
cd "C:\Users\Markus Karg\x"
copy "C:\payara41\glassfish\domains\domain1\config\glassfish-acc.xml" "C:\Users\Markus Karg\x\my-acc.xml"
package-appclient.bat -xml my-acc.xml
jar -tf appclient.jar | find "-acc.xml"
```

## Environment ##
- **Payara Version**: 4.1.2.172
- **Edition**: Full
- **JDK Version**: Oracle JDK 8u144 (64 Bit)
- **Operating System**: Windows 7 Pro SP1

**Comment by fturizo** [comment]
@mkarg, sorry for the late reply. Since you have a support contract for this issue, we are going to handle this issue on the support portal. I'll leave this issue open and reference any possible fixes or workarounds as well.

**Comment by mkarg** [comment]
Thanks for picking this up! In fact the issue is not vital for us. We just noticed that the problem exists. But if having a support contract implies that the bug will get fixed, this would certainly be great! :-)

**Comment by fturizo** [comment]
Confirmed bug, created internal JIRA Issue `PAYARA-1922` to solve the problem.

**Comment by OndrejM** [comment]
Fixed by #1877
